patches/bess/0012-ExactMatch-Bulk-Support.patch

```diff
@@ -1,15 +1,53 @@
-From 460ac23d8143f8475a7819aa8ccbbf930f2940c8 Mon Sep 17 00:00:00 2001
-From: Amar Srivastava <amarsri28@gmail.com>
-Date: Wed, 23 Dec 2020 02:33:49 -0800
-Subject: [PATCH] dpdk-hash linking and pipeline improvements
+From 54cd30fdc2cac86900d564e90118b7fbef7bef20 Mon Sep 17 00:00:00 2001
```

Comment:
No need to change the patch file name. The functionality of the patch file did not change. It still provides DPDK concurrent hash table support and pipeline improvements